packages/dbml-parse/src/lib/analyzer/binder/types.ts

```diff
@@ -23,6 +23,9 @@ export type BinderRule =
     ignoreNameNotFound: boolean;
     // A list of keywords that, when encountered by the binder as a standalone variable, skipped
     keywords?: readonly string[];
+    // In attribute value, a quoted variables can be sometimes used as a string
+    // This option decides whether it is legal for a standalone quoted variable to be treated as a string and not bound
+    ignoreNameNotFoundForQuotedVariable: boolean;
```

Comment:
In order to prevent numerous declarations in other code, should we set this one's default value to false?